azerothcore/azerothcore-wotlk
# feat(Core): add config option for LFG expansion

## Changes Proposed:
-  Add config option for dungeon finder expansion

Before this pr when you have tbc dungeons disabled a player see's this:

![image](https://user-images.githubusercontent.com/4015312/129642457-d7716233-9f79-4b81-bd45-66121b4f00c4.png)

With this pr players will see this:
![image](https://user-images.githubusercontent.com/4015312/129642512-11ed5e60-3221-4ea0-9048-74ca6c50ccbb.png)



## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Build and run
- set `DungeonFinder.Expansion` in your worldserver.conf
- test in game with different config options to see how the client shows it to the player


## How to Test the Changes:


1. set `DungeonFinder.Expansion = 0 ` in the worldserver.conf
2. build and run
3.  set character level 59 (for tbc)
4. Check in-game how the dungeon finder display's to the player 

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
